.travis.yml

```diff
@@ -4,3 +4,7 @@ go:
   - 1.7
   - 1.8
 go_import_path: google.golang.org/genproto
+
+script:
+- go test -v ./...
+- go get -v cloud.google.com/go/...
```

Comment:
oh right right duh about -u. +1

hm, i think maybe you misunderstand my suggestion?
above, with the `if` conditions i've suggested, the go-get will only run after a PR merge, only on master. the check will never be part of PR submission.
they way you have it now, the go-get check will be part of the pre-condition to merge a PR.

but i'm not completely tied to this idea. i like the idea of kicking off a cloud build better.
but if you'd rather have some manual system in place (documentation/just remember), i'm also fine with that for now. if it happens again or a few more times, we could revisit.

totally up to you.